brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
Glial morphologies have elliptical cross sections. That alone is a big change. Then the section types are completely different also. The most correct thing would even be to create a new section type. Even more, I don't even know if a single class would be enough to represent the morphologies of all types of glial cells.